### ISSUE TITLE
Core | Speed-dial issue: white toolbar in Android 5

### ISSUE THREAD

**Issue by misterjunio** [OnsenUI/OnsenUI#1863]
__Environment__

```
[Core]
    onsenui 2.1.0

[Framework]
    none

[Framework binding]
    none

[Additional libraries]
    none

[Platform]
    Mobile - Android 5.0.1

[Browser]
    Mobile - Android Chrome
```

__Encountered problem__
Touching any `speed-dial` button briefly turns the toolbar white.

__How to reproduce__
Open the speed-dial core example in Android 5 Chrome. Any speed-dial button tapped (except for some reason the main bottom-right one) places a momentary white bar over the toolbar.

**Comment by asial-matagawa** [comment]
@misterjunio 
Thank you for reporting.

I have investigated this problem and I think this is a `z-index` bug of Chrome 43 (installed in the device you used). Chrome 43 repaints outside of the `page__content`, but does not repaint `ons-toolbar` correctly. I think it is hard to find a proper workaround.

In Android 5.0 and later, Android System WebView and Chrome for Android are periodically updated, so I don't consider this as a critical glitch bug.

Closing this.
